repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/proxy/ContentLengthTest.groovy

```diff
@@ -23,12 +23,14 @@ import framework.ReposeValveTest
 import org.rackspace.deproxy.Deproxy
 import org.rackspace.deproxy.Handling
 import org.rackspace.deproxy.MessageChain
+import spock.lang.IgnoreIf
 import spock.lang.Unroll
 
 class ContentLengthTest extends ReposeValveTest {
 
-    def setupSpec() {
+    def timeBombDate = new GregorianCalendar(2017, Calendar.JANUARY, 5)
```

Comment:
Your test failed, and this is why? You can't reference instance variables in annotations, only statics